static/js/stream_muting.js

```diff
@@ -67,4 +68,14 @@ export function update_is_muted(sub, value) {
         )}'] #sub_is_muted_setting .sub_setting_control`,
     );
     is_muted_checkbox.prop("checked", value);
+
+    const row = $(`.stream-row[data-stream-id='${CSS.escape(sub.stream_id)}']`);
+    $(row).toggleClass("control-label-disabled", value);
+    if (value) {
+        $(row).find(".unmute_stream").show();
+    }
+    $(row).find("input[disabled]").prop("disabled", value);
+    $(row)
+        .find('[name="push_notifications"]')
+        .prop("disabled", !page_params.realm_push_notifications_enabled);
```

Comment:
This code is duplicated with the one in `settings_notifications.js`. We can extract this into a single function and then call it instead of this duplicated code.